arelle/plugin/validate/EDINET/rules/gfm.py

```diff
@@ -1874,3 +1843,93 @@ def rule_gfm_1_10_14(
                 msg=_("A non-empty footnote is not referenced by an element"),
                 modelObject=footnote
             )
+
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+    disclosureSystems=[DISCLOSURE_SYSTEM_EDINET],
+)
+def rule_gfm_2_5_1(
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    EDINET.EC5700W: [GFM 2.5.1] A presentation linkbase of a standard taxonomy should not be included in the DTS of an instance.
+    """
+    for modelDocument in val.modelXbrl.urlDocs.values():
+        if pluginData.isStandardTaxonomyUrl(modelDocument.uri, val.modelXbrl) or not modelDocument.type == ModelDocument.Type.SCHEMA:
+            continue
+        rootElt = modelDocument.xmlRootElement
+        for elt in rootElt.iterdescendants(XbrlConst.qnLinkLinkbaseRef.clarkNotation):
+            uri = elt.attrib.get(XbrlConst.qnXlinkHref.clarkNotation)
+            role = elt.attrib.get(XbrlConst.qnXlinkRole.clarkNotation)
+            if not role == LINKBASE_REF_URIS[LinkbaseType.PRESENTATION] or pluginData.isExtensionUri(uri, val.modelXbrl):
+                continue
```

Comment:
Would be nice to consolidate the common code between these validations into `getStandardTaxonomyExtensionLinks(linkbaseType)`